src/scripts/main.js

```diff
@@ -1,5 +1,26 @@
 'use strict';
 
 document.addEventListener('click', e => {
-  // write code here
+  const spider = document.querySelector('.spider');
+  const wall = document.querySelector('.wall');
+
+  const maxTop = wall.clientHeight - spider.offsetHeight;
+  const maxLeft = wall.clientWidth - spider.offsetWidth;
+
+  if (e.target !== wall && e.target !== spider) {
+    return 0;
```

Comment:
Return statement should be 'return;' instead of 'return 0;' since it's an event listener and the return value is not used.